entity-framework/toc.yml

```diff
@@ -569,5 +569,5 @@
             href: ef6/resources/licenses/ef6/kor.md
           - name: Russian
             href: ef6/resources/licenses/ef6/rus.md
-    - name: "⤤ EF6 API Reference"
+    - name: "EF6 API Reference <span class=\"docon docon-navigate-external\" aria-hidden=\"true\" />"
       href: https://docs.microsoft.com/dotnet/api/?view=entity-framework-6.2.0
```

Comment:
😭

![image](https://user-images.githubusercontent.com/1226749/68718818-3fe18e00-055f-11ea-8e96-0cc762842a4b.png)

Reply:
Not sure how to solve this. Perhaps just remove the icon in this case?